src/status_im/contexts/wallet/send/events_test.cljs

```diff
@@ -163,3 +181,907 @@
+                                       {:chain-id 11155111}}}
+        receiver-networks [421614 11155420]]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send        {:token-display-name token-symbol
+                                   :token              token
+                                   :receiver-networks  receiver-networks}
+                     :other-props :value}}})
+    (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui]) :send)))))
+
+(h/deftest-event :wallet/select-address-tab
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:select-address-tab "tab"}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send nil}}})
+    (is (match? expected-db (:db (dispatch [event-id "tab"]))))))
+
+(h/deftest-event :wallet/clean-send-address
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:to-address  "0x01"
+                          :recipient   {:recipient-type :saved-address
+                                        :label          "label"}
+                          :other-props :value}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :to-address)))
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :recipient)))
+  (is (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :other-props)))
+
+(h/deftest-event :wallet/clean-send-amount
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:amount      10
+                          :other-props :value}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :amount)))
+  (is (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :other-props)))
+
+(h/deftest-event :wallet/clean-disabled-from-networks
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:disabled-from-chain-ids [:optimism]
+                          :other-props             :value}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :disabled-from-chain-ids)))
+  (is (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :other-props)))
+
+(h/deftest-event :wallet/clean-from-locked-amounts
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:from-locked-amounts "value"
+                          :other-props         :value}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :from-locked-amounts)))
+  (is (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :other-props)))
+
+(h/deftest-event :wallet/disable-from-networks
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:disabled-from-chain-ids [:optimism]
+                                          :other-props             :value}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props :value}}}})
+    (is (match? expected-db (:db (dispatch [event-id [:optimism]]))))))
+
+(h/deftest-event :wallet/unlock-from-amount
```

Comment:
@VolodLytvynenko - this test is not needed any more right?

Reply:
I must admit that I don't understand.

Reply:
@J-Son89 Why not?

Reply:
@mmilad75, we disabled the functionality for locking amounts so these events won't be used in the code